boards/arm/rp2040/pimoroni-tiny2040/include/rp2040_i2cdev.h

```diff
@@ -0,0 +1,72 @@
+/****************************************************************************
+ * boards/arm/rp2040/pimoroni-tiny2040/include/rp2040_i2cdev.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __BOARDS_ARM_RP2040_PIMORONI_TINY2040_INCLUDE_RP2040_I2CDEV_H
+#define __BOARDS_ARM_RP2040_PIMORONI_TINY2040_INCLUDE_RP2040_I2CDEV_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include <stdint.h>
```

Comment:
```suggestion
 
#​ifndef​ __ASSEMBLY__
#  include <stdint.h>
#endif
```

Reply:
Is this common practice in NuttX?

Reply:
You have the same in boards/arm/rp2040/pimoroni-tiny2040/include/board.h from you PR.
In general if you expect header files to be included in assembly then having `stdint.h` will definitely bring you into troubles.